domains/tilak.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "tilaksharma-ai",
+        "email": "212199845+tilaksharma-ai@users.noreply.github.com"
```

invalid email, if you dont want to expose your email you can safely delete this
```suggestion
        "username": "tilaksharma-ai"
```